src/screens/ui/RootNavigator.tsx

```diff
@@ -0,0 +1,109 @@
+import { TouchableOpacity } from 'react-native';
+
+import { useNavigation } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { useTranslation } from 'react-i18next';
+
+import { APP_VERSION, colors, ENV } from '@app/shared/lib';
+import { Text, CloseIcon, UserProfileIcon } from '@shared/ui';
+
+import { getScreenOptions, RootStackParamList } from '../config';
+import { useDefaultRoute, useInitialRouteNavigation } from '../model';
+import {
+  AppletsScreen,
+  ChangeLanguageScreen,
+  ForgotPasswordScreen,
+  LoginScreen,
+  SignUpScreen,
+  AboutScreen,
+} from '../ui';
+
+const Stack = createNativeStackNavigator<RootStackParamList>();
+
+export default () => {
+  const { t } = useTranslation();
+  const navigation = useNavigation();
+
+  const defaultRoute = useDefaultRoute();
+
+  useInitialRouteNavigation();
+
+  return (
+    <Stack.Navigator
+      screenOptions={getScreenOptions}
+      initialRouteName={defaultRoute}>
+      <Stack.Screen
+        options={{ headerShown: false }}
+        name="Login"
+        component={LoginScreen}
+      />
+
+      <Stack.Screen
+        name="ForgotPassword"
+        options={{
+          title: t('login:forgot_password'),
+        }}
+        component={ForgotPasswordScreen}
+      />
+
+      <Stack.Screen
+        name="SignUp"
+        options={{
+          title: t('login:new_user'),
+          contentStyle: {
+            borderTopColor: colors.grey,
+            borderTopWidth: 1,
+          },
+        }}
+        component={SignUpScreen}
+      />
+
+      <Stack.Screen
+        name="ChangeLanguage"
+        options={{
+          title: t('language_screen:app_language'),
+        }}
+        component={ChangeLanguageScreen}
+      />
+
+      <Stack.Screen
+        name="AboutApp"
+        options={{
+          title: t('about_app:title_with_version', {
+            version: ENV ? `${APP_VERSION} ${ENV}` : APP_VERSION,
+          }),
+          headerTitleStyle: {
+            fontSize: 16,
+            color: colors.white,
+          },
+        }}
+        component={AboutScreen}
+      />
+
+      <Stack.Screen
+        name="Applets"
+        options={{
+          title: 'Applets',
```

Comment:
The Title should be "Hi [User Name]"

Reply:
I can do but I don't think it is related to my PR because it was not me who worked on this screen